tests/yam/agama/auto.pm

```diff
@@ -12,11 +12,12 @@ use testapi;
 use utils 'clear_console';
 
 sub run {
-    assert_screen('agama-install-finished', 1200);
+    # workaround to add needle to trust the signing key 
+    assert_screen('agama-signing-key', 600);
+    assert_and_click('trust');
```

Comment:
we should make this conditional, if the needle match, click and soft fail.

Reply:
updated to muti-tag, please have a look.